internal/state/conditions/conditions.go

```diff
@@ -294,6 +285,17 @@ func NewRouteBackendRefUnsupportedValue(msg string) Condition {
 	}
 }
 
+// NewRouteInvalidGateway returns a Condition that indicates that the Route is not Accepted because the Gateway it
+// references is invalid.
+func NewRouteInvalidGateway() Condition {
+	return Condition{
+		Type:    string(v1beta1.GatewayClassConditionStatusAccepted),
```

Comment:
should this have `v1beta1.RouteConditionAccepted` as type? and also, why not move it close to other Route-related conditions?

Reply:
Yes to RouteConditionAccepted. It is by the other Route-related conditions...

Reply:
Ah, I see what you mean. There are two groups of Route conditions. I combined them and reorganized the file a bit so everything is grouped by resource.